main/views.py

```diff
@@ -3,6 +3,8 @@
 from django.shortcuts import redirect
 from django.contrib.auth import logout
 
```

Comment:
a blank line here is unnecessary

Reply:
Pycharm added this line, after reformatting the file. I'll remove it anyway.